key/aziot-keys/aziot-keys.h

```diff
@@ -33,7 +33,7 @@ typedef unsigned int AZIOT_KEYS_VERSION;
 /**
  * The base struct of all of function lists.
  */
-typedef struct {
+typedef struct AZIOT_KEYS_FUNCTION_LIST {
```

Comment:
Why did these changes happen? Are you sure you installed the version of cbindgen listed in the install script and not the latest one? Indeed, CI is complaining because it's generating a different header than this one.

Reply:
Well, apparently I also had the distro's bindgen / cbindgen packages installed.  After removing those and ensuring my PATH pointed to the latest rust binaries the generated aziot-keys.h looks like it did before.